less/_defaults/_variables.less

```diff
@@ -79,10 +80,6 @@
 // --------------------------------------------------
 @component-padding-hoz: 0.5rem;
 
-// Navigation height
-// --------------------------------------------------
-@navigation-height: @icon-size + (@icon-padding * 2);
```

Comment:
Probably wise to keep this default in for the time being? With a deprecated comment.